src/components/v2v3/shared/SplitItem/JuiceboxProjectBeneficiary.tsx

```diff
@@ -0,0 +1,79 @@
+import { Trans } from '@lingui/macro'
+import { Space, Tooltip } from 'antd'
+import FormattedAddress from 'components/FormattedAddress'
+import TooltipLabel from 'components/TooltipLabel'
+import { NULL_ALLOCATOR_ADDRESS } from 'constants/contracts/mainnet/Allocators'
+import { FEATURE_FLAGS } from 'constants/featureFlags'
+import { Split } from 'models/splits'
+import { featureFlagEnabled } from 'utils/featureFlags'
+import { AllocatorBadge } from '../FundingCycleConfigurationDrawers/AllocatorBadge'
+import V2V3ProjectHandleLink from '../V2V3ProjectHandleLink'
+import { CrownFilled } from '@ant-design/icons'
+import { DiffedItem } from 'components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/DiffedItem'
+
+export function JuiceboxProjectBeneficiary({
+  projectOwnerAddress,
+  split,
+  oldSplit,
+}: {
+  split: Split
+  oldSplit?: Split
+  projectOwnerAddress: string | undefined
+}) {
+  if (!split.projectId) return null
+
+  const isProjectOwner = projectOwnerAddress === split.beneficiary
+  const allocatorsEnabled = featureFlagEnabled(FEATURE_FLAGS.SPLIT_ALLOCATORS)
```

Comment:
**question (non-blocking, unrelated to this PR)** are we ready to remove this feature flag now?

Reply:
Yep, we are